CHANGELOG.md

```diff
@@ -107,6 +116,15 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 ### Changed
 
+- **CUMULUS-1736**
+  - The `collections` model in the API package now determines the name of a
```

Comment:
Should we mention that items in the collection in the collection config store are also now stored by name/version instead of dataType/version?

Reply:
I don't think so, the CollectionConfigStore is an internal implementation detail of the API, and isn't exposed to the world.

Reply:
The changes in that file are purely renaming the variables, there are no functional changes in how those are stored or fetched.